railties/lib/rails/rack/logger.rb

```diff
@@ -79,6 +82,12 @@ def development?
       def logger
         Rails.logger
       end
+
+      def show_extra_line_in_development_for_non_assets_requests(path)
+        if development? && path !~ /^\/assets/
```

Comment:
Nice catch! It works with vendored assets but won't work with changing assets path.